setup.py

```diff
@@ -24,7 +24,7 @@
     classifiers=[
         "Development Status :: 4 - Beta",
```

Comment:
Might be time to update this one?

```suggestion
        "Development Status :: 5 - Production/Stable",
```